src/libraries/System.Data.Odbc/ref/System.Data.Odbc.netstandard.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the https://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Data.Odbc
+{
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
```

Comment:
This should not be added see https://github.com/dotnet/runtime/blob/master/src/libraries/System.Data.Odbc/src/MatchingRefApiCompatBaseline.txt

Reply:
Perhaps you found these as part of your GenerateReferenceSource work.  If that's the case, then maybe we should have common exclusions for reverse APICompat and GenAPI.  Either this file or some new one.